src/hooks/usePendingTxs.ts

```diff
@@ -62,6 +62,17 @@ export const usePendingTxsQueue = (): {
     // Adjust the first label ("Next" -> "Pending")
     if (results[0] && isLabelListItem(results[0])) {
       results[0].label = 'Pending' as LabelValue
+
+      if (results.filter((item) => isTransactionListItem(item)).length === 0) {
+        results.splice(0, 1)
+      }
+    }
+
+    if (results[1] && isConflictHeaderListItem(results[1])) {
+      // Check if we both conflicting txs are still pending
+      if (results.filter((item) => isTransactionListItem(item)).length <= 1) {
+        results.splice(1, 1)
+      }
```

Comment:
Not sure I understand these two conditions. If the first element is a label and there is nothing else we remove the label. Why not check for `results.length === 1` in that case?

For the second condition I don't see how it is connected to pending transactions. If we filter all of them out anyway does it matter if one or two are pending?

Reply:
You could execute the same tx twice. Once with i.e. metamask and once with e.g. ledger. So there is in theory the possibility to have conflicting pending txs. But if only one of the untrusted queue txs is pending we have to remove the conflicting tx label.